src/components/InputField/PasswordInput.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import { TextInput, View, Pressable } from 'react-native';
+import Styles from './styles';
+import Colors from '../../../assets/Colors';
+import { MaterialCommunityIcons } from '@expo/vector-icons';
+
+export const PasswordInput = (props: any) => {
+  const [isActive, setIsActive] = useState(false);
+  const { passwordVisibility, handlePasswordVisibility } =
+    togglePasswordVisibility();
+
+  return (
+    <View>
+      <View style={isActive ? Styles.FormFieldFocus : Styles.FormField}>
+        <TextInput
+          style={Styles.TextInputField}
+          onBlur={() => setIsActive(false)}
+          onFocus={() => setIsActive(true)}
+          onChangeText={newText => props.onChange(newText)}
+          value={props.value}
+          placeholder={props.placeholder}
+          placeholderTextColor={Colors.midGray}
+          secureTextEntry={passwordVisibility}
+          autoCorrect={false}
+          autoCapitalize="none"
+        />
+        <Pressable style={Styles.Icon} onPress={handlePasswordVisibility}>
+          <MaterialCommunityIcons
+            name={passwordVisibility ? 'eye-outline' : 'eye-off-outline'}
+            size={22}
+            color="#232323"
```

Comment:
This should use `Colors.darkGray`

Also `darkGray` should be `#525454`, not `#272929` -- you can go ahead and fix that.